### PR TITLE
Ensure migrations run before pulp services

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: assets/wait_on_database_migrations.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    sleep 5
+  fi
+done
+
+if [ $database_migrated = false ]; then
+  echo "Database not migrated in time, exiting"
+  exit 1
+else
+  exit 0
+fi
```
Comment (fao89) [inline assets/wait_on_database_migrations.sh]: got from pulp-operator: https://github.com/pulp/pulp-operator/blob/main/containers/images/pulp/container-assets/wait_on_database_migrations.sh

**Review by dkliban — APPROVED** [review]

File: assets/000-postgres.prep [hunk 2]
```diff
@@ -117,3 +117,7 @@ foreground {
   s6-setuidgid postgres
   /usr/local/bin/pulpcore-manager migrate
 }
+pipeline {
+  echo "ready"
+}
+tee /database/status
```
Comment (mdellweg) [inline assets/000-postgres.prep:123]: This does not seem to be used. (I would have preferred that method.)

Comment (fao89) [reply assets/000-postgres.prep:123]: it is used here: https://github.com/pulp/pulp-oci-images/blob/latest/assets/wait_on_database_migrations.sh#L11

Comment (mdellweg) [reply assets/000-postgres.prep:123]: Oh right, but that line has no consequences for any of the services.

**Pull request merged**